source/functions.txt

```diff
@@ -4,7 +4,12 @@
 Atlas Functions
 ===============
 
-.. default-domain:: mongodb
+.. meta:: 
+  :description: Atlas Functions are executable, server-side JavaScript code that define your app's behavior.
```

Comment:
🤔 

I'm not sure I'm sold on "define your app's behavior." (I know this just came from the opening sentence.) What about...

```suggestion
  :description: Define Atlas Functions and call them manually or automatically to execute server-side JavaScript code.
```